release-notes/v1_62.md

```diff
@@ -81,6 +81,24 @@ There is a new setting `html.completion.attributeDefaultValue` that lets you cho
 
 Work continues on the [GitHub Pull Requests and Issues](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github) extension, which allows you to work on, create, and manage pull requests and issues. Check out the [changelog for the 0.32.0](https://github.com/microsoft/vscode-pull-request-github/blob/main/CHANGELOG.md#0320) release of the extension to see the highlights.
 
+### Jupyter
+
+**Kernels**
+
+The list of kernels displayed in the kernel picker is now grouped to make it easier for one to identify and find kernels belonging to a specific Python envoironment.
+
+![Kernels grouped in kernel picker](images/1_62/kernel_picker.png)
+
+One can now filter the list of kernels displayed in the kernel picker. This filter can be applied per user or per workspace. This would prove to be an invaluable tool for those with large number of kernels in the kernel picker, as one can now ensure only a subset of the kernels are displayed for a specific workspace.
+
+To manage the list of kernels displayed, please use the command `Jupyter: Filter Kernels` from the command palette.
+
+![Kernel Filter](images/1_62/filter_kernels.gif)
+
+**Interactive Window**
+
+Automatic cell creation when running the last cells in an `Interactive Window` using `Shift+Enter` can now be configured via the setting `jupyter.newCellOnRunLast`.
```

Comment:
Should we thank janosh?

Reply:
They already get a mention in our chagne log, not sure we want to call out specific individuals here, wouldn't be fair to others. As there are others who contributed fixes.

Either we name all of them or none of them.
What do you think?

Reply:
Well sure others contributed fixes, but Janosh did that feature? I think we should mention his name. In other release notes are thanks added to outside users?

Reply:
Here's an example of somebody being thanked explicitly:

https://github.com/microsoft/vscode-docs/blob/0cc1d636418f7ee19eef2f03968d9aa2fee9eda2/release-notes/v1_13.md

Reply:
Done, please double chekc

Reply:
We try not to thank folks in the body of the release notes. Can you add a mention and the relevant PR to the Thank you section at the bottom of the release notes?

Reply:
You are also linking to an issue rather than a PR. Should this be PR https://github.com/microsoft/vscode-jupyter/pull/7995?

Reply:
Updated with https://github.com/microsoft/vscode-docs/commit/10a10e28a1169b6677c43919eb70f8c9c49acd15